browser-test/src/admin/northstar_admin_program_preview.test.ts

```diff
@@ -0,0 +1,51 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from '../support'
+
+test.describe('admin program preview', {tag: ['@northstar']}, () => {
+  test.beforeEach(async ({page}) => {
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+  })
+
+  test('preview program and use back button', async ({
+    page,
+    adminPrograms,
+    adminQuestions,
+    applicantQuestions,
+  }) => {
+    await loginAsAdmin(page)
+
+    await adminQuestions.addEmailQuestion({questionName: 'email-q'})
+    const programName = 'test program'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.editProgramBlockUsingSpec(programName, {
+      description: 'description',
+      questions: [{name: 'email-q'}],
+    })
+    await adminPrograms.gotoEditDraftProgramPage(programName)
+
+    await validateScreenshot(page, 'northstar-admin-edit-draft-program-page')
```

Comment:
suggestion (blocking): remove this screenshot since it's not related to the work you did. i'm pretty sure we have a screenshot of this elsewhere in our browser tests.